docs/conf.py

```diff
@@ -126,14 +128,24 @@ def django_release():
 # The "development version" of Django
 django_next_version = "4.2"
 
-extlinks = {
-    "bpo": ("https://bugs.python.org/issue?@action=redirect&bpo=%s", "bpo-"),
-    "commit": ("https://github.com/django/django/commit/%s", ""),
-    "cve": ("https://nvd.nist.gov/vuln/detail/CVE-%s", "CVE-"),
-    # A file or directory. GitHub redirects from blob to tree if needed.
-    "source": ("https://github.com/django/django/blob/main/%s", ""),
-    "ticket": ("https://code.djangoproject.com/ticket/%s", "#"),
-}
+if sphinx_version < "4.0.0":
+    extlinks = {
+        "bpo": ("https://bugs.python.org/issue?@action=redirect&bpo=%s", "bpo-"),
+        "commit": ("https://github.com/django/django/commit/%s", ""),
+        "cve": ("https://nvd.nist.gov/vuln/detail/CVE-%s", "CVE-"),
+        # A file or directory. GitHub redirects from blob to tree if needed.
+        "source": ("https://github.com/django/django/blob/main/%s", ""),
+        "ticket": ("https://code.djangoproject.com/ticket/%s", "#"),
+    }
+else:
+    extlinks = {
+        "bpo": ("https://bugs.python.org/issue?@action=redirect&bpo=%s", "bpo-%s"),
+        "commit": ("https://github.com/django/django/commit/%s", "%s"),
+        "cve": ("https://nvd.nist.gov/vuln/detail/CVE-%s", "CVE-%s"),
+        # A file or directory. GitHub redirects from blob to tree if needed.
+        "source": ("https://github.com/django/django/blob/main/%s", "%s"),
+        "ticket": ("https://code.djangoproject.com/ticket/%s", "#%s"),
+    }
```

Comment:
The new syntax is available in Sphinx 4+, see https://github.com/sphinx-doc/sphinx/commit/93cf1a57d916a1ff96c8e8a0356d0256e40489ac.

Reply:
Probably yes :disappointed: , `djangoproject.com` uses version 1.8.4, so I first have to carefully bump version on `djangoproject.com`. I will try to do this later this week.

Reply:
See:

- https://github.com/django/django/pull/15746
- https://github.com/django/djangoproject.com/pull/1169

we should be relatively save since we only rebuild docs for supported versions (see https://github.com/django/djangoproject.com/commit/f9bc2bcc296720c7a2fbc6e0c220889509e80481).